api/src/main/java/org/jboss/forge/roaster/Problem.java

```diff
@@ -106,7 +112,6 @@ else if (!message.equals(other.message))
    @Override
    public String toString()
    {
-      return "Problem [message=" + message + ", sourceStart=" + sourceStart + ", sourceEnd=" + sourceEnd
-               + ", sourceLineNumber=" + sourceLineNumber + "]";
+      return "Problem: " + message;
```

Comment:
Maybe it is interesting to show the other attributes too?

Reply:
Valid point. Will change this